src/lib/viewers/BaseViewer.js

```diff
@@ -1092,42 +1119,13 @@ class BaseViewer extends EventEmitter {
      * @return {Object} combined options
      */
     createAnnotatorOptions(moreOptions) {
-        // Temporary solution for localizing strings in the BoxAnnotations npm package
-        // TODO(@spramod): Remove once BoxAnnotations has it's own localization strategy
-        const localizedStrings = {
```

Comment:
Can we pass both for now and place the new `box-annotations-messages` import behind a try/catch or something? That way we can still merge without needing to upgrade box-annotations just yet.